docker/airflow/1.10.4/Dockerfile

```diff
@@ -55,39 +58,43 @@ RUN apk update \
 		libxml2-dev \
 		libxslt-dev \
 		linux-headers \
-		mariadb-dev \
 		nodejs \
 		nodejs-npm \
-		postgresql-dev \
 		python3-dev \
 		tzdata \
-		libsodium \
-		libsodium-dev \
 	&& apk add --no-cache \
 		bash \
 		ca-certificates \
 		cyrus-sasl \
+		cython3@edge \
+		freetds \
 		krb5-libs \
-		mariadb-connector-c \
-		postgresql \
+		py3-bcrypt@edge \
+		py3-cassandra-driver@edge-testing'>='3 \
+		py3-cryptography@edge \
+		py3-fastavro@edge-testing \
+		py3-grpcio@edge-testing \
+		py3-mysqlclient \
+		py3-numpy-dev@edge-community \
+		py3-numpy@edge-community \
+		py3-pandas@edge-testing \
+		py3-psycopg2 \
+		py3-pycryptodome@edge-community \
+		py3-pynacl@edge \
+		py3-typed-ast@edge-testing \
 		python3 \
 		tini \
 	&& update-ca-certificates \
 	&& cp /usr/share/zoneinfo/UTC /etc/localtime \
 	&& pip3 install --no-cache-dir --upgrade pip==19.2.1 \
 	&& pip3 install --no-cache-dir --upgrade setuptools==41.0.1 \
-	&& pip3 install --no-cache-dir cython \
-	&& pip3 install --no-cache-dir numpy \
-	&& pip3 install --no-cache-dir flask==1.0.0 \
-	# https://pynacl.readthedocs.io/en/stable/install/
-	&& SODIUM_INSTALL=system pip3 install --no-cache-dir pynacl==1.3.0 \
 	&& pip3 install --no-cache-dir "${AIRFLOW_MODULE}" \
 	&& pip3 install --no-cache-dir "https://github.com/astronomer/astronomer-fab-securitymanager/releases/download/v1.2.1/astronomer_fab_security_manager-1.2.1-py3-none-any.whl" \
 	&& cd /usr/lib/python3.7/site-packages/airflow/www_rbac \
 	&& npm install \
 	&& npm run build \
 	&& rm -rf node_modules \
-	&& apk del .build-deps \
+	&& apk del .build-deps py3-numpy-dev \
```

Comment:
@ashb should we add `py3-numpy-dev` to the upper half of this command as part of `.build-deps`?

Reply:
Ideally yes, but you can't put "masked" deps (the `@edge` edge is a mask) in to a virtual package